test/module/irohad/execution/command_validate_execute_test.cpp

```diff
@@ -2181,13 +2182,41 @@ TEST_F(SetAccountDetailTest, InValidWhenOtherCreator) {
  * @when creator is setting details to their account
  * @then successfully execute the command
  */
-TEST_F(SetAccountDetailTest, ValidWhenHasPermissions) {
+TEST_F(SetAccountDetailTest, ValidWhenHasRolePermission) {
   // TODO 2018-04-20 Alexey Chernyshov - IR-1276 - rework with CommandBuilder
   command = buildCommand(
       TestTransactionBuilder().setAccountDetail(kAccountId, kKey, kValue));
   set_aacount_detail =
```

Comment:
Can we fix the typo?) `set_aacount_detail` -> `set_account_detail` (since the variable was touched by the pr)